app/views/videos/search.html.haml

```diff
@@ -0,0 +1,6 @@
+
+%article.video_category
+  - @results.each do |video|
+    .video.col-sm-2
+      = link_to video_path(video) do
```

Comment:
You can use `video` shorthand for named route to build it instead of `viode_path(video)`.